<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParamSet</class>
 <widget class="QWidget" name="ParamSet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>458</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>参数设置</string>
  </property>
  <widget class="QLabel" name="labelLamda">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>波长</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxLamda">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>225</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
  </widget>
  <widget class="QLabel" name="labelLamdaUnit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>220</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>nm</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupType">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>141</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>样本类型</string>
   </property>
   <widget class="QRadioButton" name="radioRelief">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>115</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>浮雕</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioRefraction">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>115</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>折射率</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="labelRefraction">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>357</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>样本折射率</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelThick">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>220</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>样本厚度</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelThickUnit">
   <property name="geometry">
    <rect>
     <x>403</x>
     <y>220</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>um</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxThick">
   <property name="geometry">
    <rect>
     <x>335</x>
     <y>225</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineRefraction">
   <property name="geometry">
    <rect>
     <x>108</x>
     <y>360</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelSpace">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>266</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>摄像机像素间隔</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxSpace">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>270</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelScale">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>262</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>放大倍数</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxScale">
   <property name="geometry">
    <rect>
     <x>335</x>
     <y>266</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelName">
   <property name="geometry">
    <rect>
     <x>228</x>
     <y>355</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>样本名称</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditName">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>360</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupTest">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>120</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>程序测试</string>
   </property>
   <widget class="QRadioButton" name="radioTestNo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>71</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>否</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioTestYes">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>81</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>是</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupReverse">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>50</y>
     <width>120</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>像素反转</string>
   </property>
   <widget class="QRadioButton" name="radioReverseNo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>71</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>否</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioReverseYes">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>81</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>是</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnOk">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>430</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>430</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
